src/ticketmap/index.tsx

```diff
@@ -151,7 +151,7 @@ const $costRanges = createDeepEqualSelector(
   }
 )
 
-const areSetsEqual = (setA, setB) => {
+const areSetsEqual = (setA: Set, setB: Set): boolean => {
```

Comment:
Just curious: why should we create a custom comparison when lodash has [`isEqual`](https://lodash.com/docs/4.17.11#isEqual)?